static/app/views/preprod/buildDetails/main/insights/alternativeIconsInsightInfoModal.tsx

```diff
@@ -0,0 +1,156 @@
+    local output="$(basename "$input" | sed 's/\\.[^.]*$//')_optimized.png"
+
+    [ ! -f "$input" ] && echo "❌ File not found: $input" && return 1
+
+    echo "🔄 Optimizing $(basename "$input")..."
+
+    # Resize: original → 180px → 1024px (simulates homescreen quality)
+    sips --resampleWidth 180 "$input" --out /tmp/icon.png >/dev/null 2>&1 || return 1
+    sips --resampleWidth 1024 /tmp/icon.png --out "$output" >/dev/null 2>&1
+
+    rm /tmp/icon.png
+
+    if [ -f "$output" ]; then
+        local saved=$(( ($(stat -f%z "$input") - $(stat -f%z "$output")) / 1024 ))
+        echo "✅ Saved \${saved}KB → $output"
+    else
+        echo "❌ Optimization failed"
+        return 1
+    fi
+}`;
+
+type OutputFormat = 'heic' | 'png';
+
+function AlternativeIconsContent() {
+  const [outputFormat, setOutputFormat] = useState<OutputFormat>('heic');
+
+  const currentScript = outputFormat === 'heic' ? HEIC_SCRIPT : PNG_SCRIPT;
+  const exampleCommand = 'optimize_icon YourImage.png';
+
+  const description =
+    outputFormat === 'heic'
+      ? t(
+          'Use this script to optimize your images locally. It reduces file sizes by simulating homescreen quality (180px → 1024px) and converts to HEIC format.'
+        )
+      : t(
+          'Use this script to optimize your images locally. It reduces file sizes by simulating homescreen quality (180px → 1024px) and keeps them as PNG files.'
+        );
+
+  return (
+    <Fragment>
+      <Text>{description}</Text>
+
+      <Container padding="md 0">
+        <RadioGroup
+          label={t('Output format')}
+          value={outputFormat}
+          choices={[
+            ['heic', t('Convert to HEIC')],
+            ['png', t('Optimize PNG')],
+          ]}
+          onChange={(value: string) => setOutputFormat(value as OutputFormat)}
+          orientInline
+        />
+      </Container>
+
+      {outputFormat === 'heic' && (
+        <Alert type="warning">
+          {t(
+            "Reminder: If you convert your image to HEIC, make sure to update the reference in your app's project to use this new filepath!"
+          )}
+        </Alert>
+      )}
+
+      <CodeBlockWrapper>
+        <CodeBlock language="bash" filename="optimize.sh">
+          {currentScript}
+        </CodeBlock>
+      </CodeBlockWrapper>
+
+      <Flex direction="column" gap="sm">
+        <Heading as="h3" size="md">
+          {t('How to use:')}
+        </Heading>
+        <OrderedList>
+          <li>
+            <Text>
+              {t('Save the script as')} <Code>optimize.sh</Code>
+            </Text>
+          </li>
+          <li>
+            <Text>
+              {t('Run:')} <Code>source optimize.sh</Code>
+            </Text>
+          </li>
+          <li>
+            <Text>
+              {t('Optimize your images:')} <Code>{exampleCommand}</Code>
+            </Text>
+          </li>
+        </OrderedList>
+      </Flex>
+    </Fragment>
+  );
+}
+
+export function openAlternativeIconsInsightModal() {
```

Comment:
Would recommend just having the `openInsightInfoModal` be inline in `AppSizeInsightsSidebarRow`, this function seems unnecessary to just be a wrapper of `openInsightInfoModal`